webpack/ForemanTasks/Components/TaskDetails/Components/Task.js

```diff
@@ -0,0 +1,199 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import TaskInfo from './TaskInfo';
+import { ClickConfirmation } from '../../common/ClickConfirmation';
+
+class Task extends Component {
+  taskProgressToggle = () => {
+    const {
+      timeoutId,
+      refetchTaskDetails,
+      id,
+      state,
+      fetchTaskDetails,
+      loading,
+    } = this.props;
+    if (state === 'stopped') {
+      fetchTaskDetails(id);
+    } else if (this.props.timeoutId) {
+      this.props.taskReloadStop(timeoutId);
+    } else {
+      this.props.taskReloadStart(timeoutId, refetchTaskDetails, id, loading);
+    }
+  };
+
+  render() {
+    const {
+      taskReload,
+      externalId,
+      id,
+      state,
+      allowDangerousActions,
+      resumable,
+      cancellable,
+      hasSubTasks,
+      parentTask,
+      showUnlockModal,
+      showForceUnlockModal,
+      toggleUnlockModal,
+      toggleForceUnlockModal,
+    } = this.props;
+    const modalUnlock = (
+      <ClickConfirmation
+        showModal={showUnlockModal}
+        title={__('Unlock')}
+        body={__(
+          "This will unlock the resources that the task is running against. Please note that this might lead to inconsistent state and should be used with caution, after making sure that the task can't be resumed."
+        )}
+        confirmationMessage={__(
+          'I understand that this may cause harm and have working database backups of all backend services.'
+        )}
+        confirmAction={__('Unlock')}
+        path={`/foreman_tasks/tasks/${id}/unlock`}
+        confirmType="warning"
+        closeModal={toggleUnlockModal}
+      />
+    );
+
+    const modalForceUnlock = (
+      <ClickConfirmation
+        showModal={showForceUnlockModal}
+        title={__('Force Unlock')}
+        body={__(
+          'Resources will be unlocked and will not prevent other tasks from being run. As the task might be still running, it should be avoided to use this unless you are really sure the task got stuck'
+        )}
+        confirmationMessage={__(
+          'I understand that this may cause harm and have working database backups of all backend services.'
+        )}
+        confirmAction={__('Force Unlock')}
+        path={`/foreman_tasks/tasks/${id}/force_unlock`}
+        confirmType="danger"
+        closeModal={toggleForceUnlockModal}
+      />
+    );
+    return (
+      <React.Fragment>
+        {modalUnlock}
+        {modalForceUnlock}
+        <Grid>
+          <Row>
+            <Col xs={12}>
+              <Button
+                hidden={!allowDangerousActions}
+                className={`reload-button ${taskReload ? 'spin' : ''}`}
+                bsSize="small"
+                onClick={this.taskProgressToggle}
+              >
+                {__(`${taskReload ? 'Stop' : 'Start'}  auto-reloading`)}
+              </Button>
+              <Button
+                bsSize="small"
+                href={`/foreman_tasks/dynflow/${externalId}`}
```

Comment:
For me this points to `/foreman_tasks/dynflow/`, even if I patch the details template to return the external id.